### DISCUSSION TITLE
Log in issues on new install--documenting for posterity

### DISCUSSION THREAD

**Discussion by sandman98321** [sebr/bhyve-home-assistant#254] [General]
Hope this is the right place for this.  I don't want to open an issue, because it doesn't seem to be a problem with the HASS integration, rather suspected weirdness with how b-hyve does authentication (or with me being fat-fingered, but I don't think so...?)

I set up a new b-hyve timer (4 valve, 24634) via bluetooth over the weekend.  Then installed the hub (gen 2) this morning.

Tried to get the hub set up in HASS this morning.  Install via HACS, restart HASS.  Try to add the integration, errors when trying to log in.  Restart the whole HASS VM, just in case.  Same issue.

Thinking "authentication error, troubleshooting step #1-reset password"...I tried resetting my password through the b-hyve app, then the integration logged in first try, no problem.

The issue is that I am 99% sure I was using the right password (I was using the password saved from when I installed the app) and this wasn't an ID10T error.

I wonder if setting up the timer via bluetooth first, then setting up the hub afterward, there was some kind of out-of-sync password issue or flag on their authentication side of things (but I don't know how that's all integrated on their end).  

If you tell me "nah, I've set them up like that no problem", I'll delete this and accept it was a me problem.

**Comment by Phatman113** [comment]
Did you resolve this issue?  I've got an old and functional BHyve setup, and when I try to add it to HA, I'm just getting a "Failed to connect" error

**Reply by sandman98321** [reply]
IIRC, I resolved it by doing a factory reset on everything, setting up the hub, then setting up the timer.  I think my issue was caused by having set up the timer before the hub...either BHYVE or HASS didn't seem to appreciate that.
